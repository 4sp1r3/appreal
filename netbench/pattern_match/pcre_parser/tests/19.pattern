/pcre[a-z0-9\D]?[\w]*pc+re/