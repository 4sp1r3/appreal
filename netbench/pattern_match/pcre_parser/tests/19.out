lex	: modif_front: SLASH
yacc	: pcre_delim <<< SLASH
yacc	: modif_front <<< pcre_delim
lex	: ASCII p
yacc	: element <<< ASCII p
yacc	: unit <<< element
lex	: ASCII c
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII c
yacc	: unit <<< element
lex	: ASCII r
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII r
yacc	: unit <<< element
lex	: ASCII e
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII e
yacc	: unit <<< element
lex	: LBOX
yacc	: ext_unit <<< unit
yacc	: classStart <<< LBOX
lex	: CHARCLASS2VALUE char-char
lex	: a-z
yacc	: rangechars <<< INT 31329
yacc	: inclass_unit <<< rangechars
lex	: CHARCLASS2VALUE char-char
lex	: 0-9
yacc	: rangechars <<< INT 14640
yacc	: inclass_unit <<< rangechars
lex	: NDECDIGIT
yacc	: slashcharclass <<< NDECDIGIT
yacc	: inclass_element <<< slashcharclass
yacc	: inclass_unit <<< inclass_element
lex	: RBOX
yacc	: inclass_ext_unit <<< inclass_unit
yacc	: inclass_ext_unit <<< inclass_unit-inclass_ext_unit
yacc	: inclass_ext_unit <<< inclass_unit-inclass_ext_unit
yacc	: inclass <<< inclass_ext_unit
yacc	: classEnd <<< RBOX
yacc	: class <<< classStart inclass classEnd
yacc	: unit <<< class
lex	: ZEROONE
yacc	: quantifier <<< ZEROONE
lex	: LBOX
yacc	: quantify <<< quantifier
yacc	: quantify_unit <<< unit quantify
yacc	: ext_unit <<< quantify_unit
yacc	: classStart <<< LBOX
lex	: WORDCHAR
yacc	: slashcharclass <<< WORDCHAR
yacc	: inclass_element <<< slashcharclass
yacc	: inclass_unit <<< inclass_element
lex	: RBOX
yacc	: inclass_ext_unit <<< inclass_unit
yacc	: inclass <<< inclass_ext_unit
yacc	: classEnd <<< RBOX
yacc	: class <<< classStart inclass classEnd
yacc	: unit <<< class
lex	: ZEROMORE
yacc	: quantifier <<< ZEROMORE
lex	: ASCII p
yacc	: quantify <<< quantifier
yacc	: quantify_unit <<< unit quantify
yacc	: ext_unit <<< quantify_unit
yacc	: element <<< ASCII p
yacc	: unit <<< element
lex	: ASCII c
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII c
yacc	: unit <<< element
lex	: ONEMORE
yacc	: quantifier <<< ONEMORE
lex	: ASCII r
yacc	: quantify <<< quantifier
yacc	: quantify_unit <<< unit quantify
yacc	: ext_unit <<< quantify_unit
yacc	: element <<< ASCII r
yacc	: unit <<< element
lex	: ASCII e
yacc	: ext_unit <<< unit
yacc	: element <<< ASCII e
yacc	: unit <<< element
lex	: SLASH
yacc	: ext_unit <<< unit
yacc	: exp <<< ext_unit
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: exp <<< ext_unit-exp
yacc	: ext_exp <<< exp
yacc	: rv <<< ext_exp
yacc	: inslash <<< rv
yacc	: pattern <<< inslash
yacc	: pcre_delim <<< SLASH
yacc	: modif_rear <<< pcre_delim
yacc	: pcre <<< modif_front pattern modif_rear
